integration_tests/e2e/activities/editActivitySchedulePrisonRegime_2weekly.cy.ts

```diff
@@ -0,0 +1,143 @@
+import getActivity from '../../fixtures/activitiesApi/getActivity.json'
+import getPrisonRegime from '../../fixtures/activitiesApi/getPrisonRegime.json'
+import moorlandIncentiveLevels from '../../fixtures/incentivesApi/getMdiPrisonIncentiveLevels.json'
+import ViewActivityPage from '../../pages/createActivity/viewActivity'
+import CustomTimesChangeOptionPage from '../../pages/createSchedule/customTimesChangeOption'
+import DaysAndTimesPage from '../../pages/createSchedule/daysAndTimes'
+import SessionTimesPage from '../../pages/createSchedule/sessionTimes'
+import Page from '../../pages/page'
+
+context('Edit activity', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.signIn()
+
+    const getActivity4 = { ...getActivity }
+    getActivity4.id = 4
+    getActivity4.schedules[0].usePrisonRegimeTime = true
+    getActivity4.schedules[0].scheduleWeeks = 2
+    getActivity4.schedules[0].slots = [
+      {
+        id: 2051,
+        timeSlot: 'AM',
+        weekNumber: 1,
+        startTime: '08:30',
+        endTime: '11:45',
+        daysOfWeek: ['Mon', 'Tue', 'Wed', 'Thu'],
+        mondayFlag: true,
+        tuesdayFlag: true,
+        wednesdayFlag: true,
+        thursdayFlag: true,
+        fridayFlag: false,
+        saturdayFlag: false,
+        sundayFlag: false,
+      },
+      {
+        id: 2052,
+        timeSlot: 'PM',
+        weekNumber: 2,
+        startTime: '13:45',
+        endTime: '16:45',
+        daysOfWeek: ['Mon', 'Tue', 'Wed'],
+        mondayFlag: true,
+        tuesdayFlag: true,
+        wednesdayFlag: true,
+        thursdayFlag: false,
+        fridayFlag: false,
+        saturdayFlag: false,
+        sundayFlag: false,
+      },
+    ]
+
+    const inmateDetails = [
+      {
+        prisonerNumber: 'A9477DY',
+        firstName: 'JOHN',
+        lastName: 'JONES',
+      },
+      {
+        prisonerNumber: 'G4793VF',
+        firstName: 'JACK',
+        lastName: 'SMITH',
+      },
+    ]
+    cy.stubEndpoint('GET', '/incentive/prison-levels/MDI', moorlandIncentiveLevels)
+    cy.stubEndpoint('GET', '/prison/prison-regime/MDI', getPrisonRegime)
+    cy.stubEndpoint('GET', '/activities/4/filtered', getActivity4)
+    cy.stubEndpoint('PATCH', '/activities/MDI/activityId/4', getActivity4)
+    cy.stubEndpoint('POST', '/prisoner-search/prisoner-numbers', inmateDetails)
+  })
+  it('changing days/sessions if using prison regime times - removing a day', () => {
```

Comment:
Update the test description?

`It('should add a day to the schedule when updating from the default regime times')`

`It('should remove a day from the schedule when updating from the default regime times')`

Reply:
👍